files/kubelet-config.json

```diff
@@ -25,6 +25,7 @@
   "hairpinMode": "hairpin-veth",
   "cgroupDriver": "cgroupfs",
   "cgroupRoot": "/",
+  "eventRecordQPS": 0,
```

Comment:
```suggestion
```

Nope. This would be unlimited, which the CIS Benchmark warns against.

Unfortunately, the CIS Benchmark is terribly confused in what it _does_ recommend, but that is its problem.

To understand the current snafu, start by looking at the source code that implemented the deprecated `--event-qps` flag. It is this flag to which the CIS Benchmark recommendation originally applied, and for this flag a setting of 0 would be interpreted the same as if the flag were omitted, and the effective QPS would be 5, which is the default value for eventRecordQPS.

Then the flag was deprecated.

Unfortunately, setting eventRecordQPS=0 (wrong!) is explicitly disabling any rate limiting, whereas omitting eventRecordQPS will result in the default value 5.